python/llm/test/run-llm-inference-tests.sh

```diff
@@ -18,10 +18,6 @@ export OMP_NUM_THREADS=$THREAD_NUM
 python -m pytest -s ${LLM_INFERENCE_TEST_DIR}/test_transformers_api.py -v
 python -m pytest -s ${LLM_INFERENCE_TEST_DIR}/test_optimize_model_api.py -v
 
-python -m pip install transformers==4.34.0
```

Comment:
Does the testing for Transformers 4.34 is no longer necessary?

Reply:
should be, mistral is put in transformers 4.36, which replaces 4.34

Reply:
yes, the testing for Transformers 4.34 is no longer necessary, since I move mistral test into general test cases.